packages/pixeloven/tasks/package.json

```diff
@@ -26,9 +26,9 @@
     "prettier": "1.18.2",
     "rimraf": "2.6.3",
     "sinon": "7.4.1",
-    "stylelint": "10.1.0",
-    "stylelint-config-recommended-scss": "3.3.0",
-    "stylelint-scss": "3.9.3",
+    "stylelint": "11.1.1",
```

Comment:
Took forever to figure out that I needed to upgrade these here in tasks to get the newer at rules to work in the example app. Not sure why that was, but my guess is that it was defaulting to the lower version listed here instead f the higher versions I listed in the app. After updating these I removed the additional deps in the app and everything seems to be working fine.

Reply:
Yeah we actually should remove the version in the app.